homeassistant/components/drop_connect/switch.py

```diff
@@ -42,7 +42,7 @@ class DROPSwitchEntityDescription(SwitchEntityDescription):
     """Describes DROP switch entity."""
 
     value_fn: Callable[[DROPDeviceDataUpdateCoordinator], int | None]
-    set_fn: Callable
+    set_fn: Callable[[Any, Any], Any]
```

Comment:
```suggestion
    set_fn: Callable[[DROPDeviceDataUpdateCoordinator, str], None]
```
Isn't this better?

Reply:
@joostlek  Yes, but mypy blows up if the return value is not `Any`.  Check out my changes in 274b76d3ff0a8ee054b85f76338262ecca36d699 and see what you think.  Not trying to make excuses, but I'm an embedded C developer and I'm still learning the subtleties of Python... Thanks for the guidance.

Reply:
I'm not accusing you of making excuses, you can just reply that mypy is annoying and I get that :).

I just now see that this was in the switch.py. Does mypy also complain on the select one?

Reply:
Yes, they both need to declare a return value of `Any` to keep mypy happy.  The suggestion you made was for select.py, but switch.py has the same declaration so I was trying to keep them in sync.  I just saw your later comment about modifying the switch.py declaration in a separate PR, so I'll remove that change from this one.

Reply:
Oh, when the changes are this small I don't mind a little code cleanup :)

Reply:
https://github.com/home-assistant/core/pull/106314

Reply:
Fixed in a019ea53b7232b2ebcdf8c19d1c787f1a7f52ab1.

Reply:
Oh we already made a separate PR for it so you would not have to do it